it-it/javascript-it.html.markdown

```diff
@@ -0,0 +1,614 @@
+---
+language: javascript
+contributors:
+    - ["Adam Brenecki", "http://adam.brenecki.id.au"]
+    - ["Ariel Krakowski", "http://www.learneroo.com"]
+filename: javascript.js
+---
```

Comment:
Please add this line to the YAML config header -
```
lang: it-it
```

Reply:
Additionally, you could also credit yourself for the contribution by adding the following lines to the YAML header after the contributors names -
```
translators:
    - ["vinniec", "https://github.com/vinniec"]
```

You could change the name and link to anything you wish as per your convenience, I just put in your GitHub details as a placeholder.